src/app/app-routing.module.ts

```diff
@@ -9,6 +9,7 @@ import { AddNewPostComponent } from './components/authorized/add-new-post/add-ne
 import { AuthGuard } from './guards/auth.guard';
 import { WelcomePageComponent } from './components/welcome-page/welcome-page.component';
 import { AuthorizedComponent } from './components/authorized/authorized.component';
+import { PostDetailsComponent } from './components/authorized/post-details/post-details.component';
```

Comment:
This is weird. Is this working? Until now I thought single quotes don't support multiline. Could be working because it's just a prettier fix or something?

Reply:
🤷‍♂️ I see it as a single line.

Reply:
'Twas GitHub formatting, not code 💢